indigo/distribution.yaml

```diff
@@ -1,6 +1,6 @@
 %YAML 1.1
 # ROS distribution file
-# see REP 143: http://ros.org/reps/rep-0143.html
+# see REP 141: http://ros.org/reps/rep-0141.html
```

Comment:
Your version of `python-rosdistro` is out of date. Please update to the latest release. For this PR manually change the patch to not modify this comment line.